qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,511 @@
+﻿from qcodes import Instrument
```

Comment:
It is recommended to use the full path for qcodes modules:
from qcodes.instrument,base import Instrument